packages/actor-sparql-parse-algebra/test/ActorSparqlParseAlgebra-test.ts

```diff
@@ -43,8 +43,11 @@ describe('ActorSparqlParseAlgebra', () => {
       return expect(actor.test({ query: 'a', queryFormat: 'sparql' })).resolves.toBeTruthy();
     });
 
-    it('should run', () => {
-      return expect(actor.run({ query: "SELECT * WHERE { ?a a ?b }" })).resolves.toMatchObject(
+    it('should run', async () => {
+      const result = await actor.run({ query: "SELECT * WHERE { ?a a ?b }" });
+
+      // TODO: I am unable to find why the match only works like this
```

Comment:
Maybe because of the `equals` methods in the quads and terms?

It may actually be good to use a dedicated matcher for these tests, like here: https://github.com/RubenVerborgh/SPARQL.js/blob/master/test/matchers/toEqualParsedQuery.js
But it may require quite some effort, so not sure if this should be done for this PR...

(I'll look into more detail at this PR later)